OpenFreeSchools.API/appsettings.json

```diff
@@ -7,6 +7,6 @@
 		"ApiKeys": "app-key"
 	},
 	"ConnectionStrings": {
-		"DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=ofs;Integrated Security=true;TrustServerCertificate=True"
+		"DefaultConnection": "Server=localhost,57000;Database=ofs;TrustServerCertificate=True;User=sa;Password=16May2023"
```

Comment:
Changed the app config, could you revert this, we'll sort this out so it doesnt need to be modified